products/BellHybrid/services/desktop/endpoints/quotes/QuotesHelper.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2017-2025, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/blob/master/LICENSE.md
+
+#include <endpoints/quotes/QuotesHelper.hpp>
+#include <endpoints/Context.hpp>
+#include <Service/Service.hpp>
+#include <service-db/DBServiceAPI.hpp>
+#include <log/log.hpp>
+
+namespace sdesktop::endpoints
+{
+    namespace json::quotes
```

Comment:
I'd move those constants to `module-services/service-desktop/endpoints/include/endpoints/JsonKeyNames.hpp`

Reply:
I saw that some of the definitions are in this file _JsonKeyNames.hpp_ and in some modules it is directly in them. I can move it to this file.

Reply:
IMO it's better to keep all these definitions in this file, just to be consistent :wink: